### PR TITLE
DOCS-2781: QA drive a rover flutter app 

### PR DESCRIPTION
**Pull request by sguequierre**
* tested on rover rental bot and does appear to work if a bit flaky, definitely produces a square-like movement and hypothetically the call is the exact same as all the others that do work.
* paired with @kevin49999 and updated code for latest SDK version. Involved readding .id but we kept my other change with isloading.

### CODE REVIEW

**Review by JessamyT — COMMENTED** [review]
If we still don't know why things are the way that they are here, maybe @clintpurser or @Oabraham1 can provide SME review/help understand the flakiness

File: docs/tutorials/control/flutter-app.md [hunk 1]
```diff
@@ -69,6 +69,12 @@ For this walkthrough, you only need to install the following:
   - When prompted, do install Cocoapods.
     You need it to support the iOS simulator.
 
+{{% alert title="Flutter version" color="note" %}}
+We recommended using Flutter 3.19.6, as this sample app was tested with this version.
+`fvm` is a useful tool for targeting specific flutter versions.
+You can run `fvm use 3.19.6` to target Flutter 3.19.6.
```
Comment (JessamyT) [inline docs/tutorials/control/flutter-app.md:75]: As a naive user, I'm not sure when/where I'd run this command. Is it after a certain step in this guide? Is it every time I build or run the app?

Comment (sguequierre) [reply docs/tutorials/control/flutter-app.md:75]: not every time but you can run it whenever before building the app. I'm honestly not sure if this needs to be clarified for most of our target audience (at this point more experienced developers) but I'll add a sentence

Comment (sguequierre) [inline docs/tutorials/control/flutter-app.md:75]:
```suggestion
You can run `fvm use 3.19.6` in the terminal before building your sample app to target Flutter 3.19.6.
```

File: docs/tutorials/control/flutter-app.md [hunk 2]
```diff
@@ -69,6 +69,12 @@ For this walkthrough, you only need to install the following:
   - When prompted, do install Cocoapods.
     You need it to support the iOS simulator.
 
+{{% alert title="Flutter version" color="note" %}}
+We recommended using Flutter 3.19.6, as this sample app was tested with this version.
```
Comment (JessamyT) [inline docs/tutorials/control/flutter-app.md:73]:
```suggestion
We recommend using Flutter 3.19.6, as this sample app was tested with this version.
```

**Review by JessamyT — APPROVED** [review]
LGTM % nit

**Pull request merged**